internals/secrethub/signup.go

```diff
@@ -181,7 +185,42 @@ func (cmd *SignUpCommand) Run() error {
 	}
 
 	cmd.progressPrinter.Stop()
+	fmt.Fprintln(cmd.io.Stdout(), "Created your account.")
+
+	createWorkspace := cmd.org != ""
+	if !createWorkspace {
+		createWorkspace, err = ui.AskYesNo(cmd.io, "Do you want to create a shared workspace for your team?", ui.DefaultYes)
+		if err != nil {
+			return err
+		}
+		if !createWorkspace {
+			fmt.Fprintln(cmd.io.Stdout(), "You can create a shared workspace later using `secrethub org init`.")
+		}
+	}
+	if createWorkspace {
+		if cmd.org == "" {
+			cmd.org, err = ui.AskAndValidate(cmd.io, "Workspace name for your team (e.g. your company name): ", 2, api.ValidateOrgName)
+			if err != nil {
+				return err
+			}
+		}
+		if cmd.orgDescription == "" {
+			cmd.orgDescription, err = ui.AskAndValidate(cmd.io, "A description (max 144 chars) for your team workspace so others will recognize it:\n", 2, api.ValidateOrgDescription)
+			if err != nil {
+				return err
+			}
+		}
+
+		_, err := client.Orgs().Create(cmd.org, cmd.orgDescription)
+		if err == api.ErrOrgAlreadyExists {
+			fmt.Fprintf(cmd.io.Stdout(), "The workspace %s already exists. If it is your organization, ask a colleague to invite you to the workspace. You can also create a new one using `secrethub org init`.\n", cmd.org)
+		} else if err != nil {
+			return err
+		} else {
+			fmt.Fprintln(cmd.io.Stdout(), "Created your shared workspace.")
+		}
+	}
+	fmt.Fprintf(cmd.io.Stdout(), "\nTo read your first secret, run:\n\n    secrethub read %s\n\n", secretPath)
```

Comment:
Do we want to say here: `Setup complete. To read your first secrets ...`?

Reply:
We either have:
```
Created your account.
Do you want to create a shared workspace for your team? (Y/n) no

To read your first secret, run:

secrethub read your-username/start/hello
```
or:
```
Created your account.
Do you want to create a shared workspace for your team? (Y/n) yes
Workspace name for your team (e.g. your company name): company-name
A description (max 144 chars) for your team workspace so others will recognize it:
This is my company.
Created your shared workspace.

To read your first secret, run:

secrethub read your-username/start/hello
```

Yes, maybe that makes sense to add, to indicate the shared workspace was the last step :+1:

Reply:
Yes and add a white line between the question block, so it becomes something like:
```
Created your account.
Do you want to create a shared workspace for your team? (Y/n) yes

Workspace name for your team (e.g. your company name): company-name
A description (max 144 chars) for your team workspace so others will recognize it:
This is my company.

Created your shared workspace.
To read your first secret, run:

secrethub read your-username/start/hello
```